src/main/resources/assets/vulkanmod/lang/ru_ru.json

```
@@ -0,0 +1,40 @@
+{
+  "vulkanmod.options.unknown": "Неизвестно",
+
+  "vulkanmod.options.pages.video": "Видео",
```

Тут находятся настройки, связанные непосредственно с экраном и представлением окна на нём. Предлагаю такой вариант:

```suggestion
  "vulkanmod.options.pages.video": "Экран",
```